miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplayTest.kt

```diff
@@ -24,24 +26,26 @@ import org.junit.runner.RunWith
 import org.mockito.Mockito
 import kotlin.test.assertTrue
 
+@ExperimentalCoroutinesApi
 @RunWith(AndroidJUnit4::class)
 class RealMiniAppDisplayTest {
     private lateinit var context: Context
-    private lateinit var basePath: String
+    private lateinit var mBasePath: String
```

Comment:
Let's keep this as `basePath`. We aren't using hungarian notation anywhere else, so we need to stay consistent.